data/korath/korath jobs.txt

```diff
@@ -243,3 +243,593 @@ mission "Kor Efreti to Wanderers: Farming"
+	clearance "You read the cargo serial numbers and the authorities give you landing coordinates."
+	cargo "New Tortuga trade" 5 10 0.8
+	illegal 50000
+	deadline 7
+	source "New Tortuga"
+	destination "New Tortuga"
+	stopover "La'ab Bamen"
+	to offer
+		has "event: know about Korath fish jobs"
+		not "New Tortuga Fish Trade (small): active"
+		"reputation: Korath" > -3000
+	on stopover
+		dialog "You drop off your <tons> of illicit fish, and receive <tons> of unknown cargo."
+	on complete
+		payment 25000 500
+		dialog "An inventory officer meets you to trade your cargo for <payment>."
+		"reputation: Korath" += 2
+
+mission "New Tortuga Fish Trade (medium)"
+	name "New Tortuga fish trade"
+	job
+	repeat
+	description "Take <tons> of fish to <stopovers> and return with <tons> of supplies for <payment>. Avoid getting caught or you may be fined."
+	clearance "You read the cargo serial numbers and the authorities give you landing coordinates."
+	cargo "New Tortuga trade" 15 50 0.8
+	illegal 100000
+	deadline 7
+	source "New Tortuga"
+	destination "New Tortuga"
+	stopover "La'ab Bamen"
+	to offer
+		random < 20
+		has "event: know about Korath fish jobs"
+		not "New Tortuga Fish Trade (medium): active"
+		"cargo space" > 20
+		"reputation: Korath" > -3000
+	on stopover
+		dialog "You drop off your <tons> of illicit fish, and receive <tons> of unknown cargo."
+	on complete
+		payment 35000 400
+		dialog "An inventory officer meets you to trade your cargo for <payment>."
+		"reputation: Korath" += 4
+
+mission "New Tortuga Fish Trade (bulk)"
+	name "New Tortuga fish trade"
+	job
+	repeat
+	description "Take <tons> of fish to <stopovers> and return with <tons> of supplies for <payment>. Avoid getting caught or you may be fined."
+	clearance "You transmit the cargo serial numbers and the authorities give you landing coordinates."
+	cargo "New Tortuga trade" 40 300 0.8
+	illegal 150000
+	deadline 7
+	source "New Tortuga"
+	destination "New Tortuga"
+	stopover "La'ab Bamen"
+	to offer
+		random < 5
+		has "event: know about Korath fish jobs"
+		not "New Tortuga Fish Trade (bulk): active"
+		"cargo space" > 50
+		"reputation: Korath" > -3000
+	on stopover
+		dialog "You drop off your <tons> of illicit fish, and receive <tons> of unknown cargo."
+	on complete
+		payment 45000 350
+		dialog "An inventory officer meets you to trade your cargo for <payment>."
+		"reputation: Korath" += 7
+
+mission "Korath: Retrieve livestock [0]"
+	name "Livestock from <stopovers>"
+	job
+	repeat
+	description "Take <tons> of trade goods to <stopovers> to trade for <tons> of unique livestock and return to <planet> for <payment>."
+	clearance "You transmit the cargo serial numbers and the authorities give you landing coordinates."
+	cargo "livestock trade" 1 4 0.8
+	source "La'ab Bamen"
+	stopover
+		government "Republic" "Pirate" "Free Worlds" "Neutral" "Independent"
+		attributes farming
```

Comment:
Would it be possible to simplify these location filters? Something like:
```
		attributes farming
		near "Sol" 0 100
```

Reply:
I tried simplifying filters at one point and was told *not* to, so... seems like a matter of style to me.